gateway/1.4/modules/ROOT/pages/flex-gateway-securing-apis.adoc

```diff
@@ -0,0 +1,20 @@
+= Securing and Governing Flex Gateway APIs
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Policies enforce rules when processing requests. This enables you to secure and govern your Flex Gateway APIs.
```

Comment:
```suggestion
Policies enforce rules when Flex Gateway processes requests. This enables you to secure and govern your Flex Gateway APIs.
```

Reply:
This <what>? 
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-%22it,%22%22this,%22%22these,%22andsoon

Reply:
Good call on "this". Fixed.